agentliveness/agent.py

```diff
@@ -145,6 +148,28 @@ def _check_cinder(self):
 
         return 0
 
+    def _check_manila(self):
+        manila = manila_client.Client(session=self._get_session())
+        try:
+            for backend in self.CONF.enabled_share_backends:
+                if backend is not None:
+                    host = "{}@{}".format(self.CONF.host, backend)
+                else:
+                    host = self.CONF.host
+                for service in manila.services.list({'host': host}):
+                    if service.state == 'up':
+                        return 0
```

Comment:
Do I understand this correctly: If the service for a single backend is up, the whole pod is fine? I see the point to restart the pod if one service is down, but not that a single service being up should determine the state of all services.